metadatastore/odm_templates.py

```diff
@@ -184,4 +165,4 @@ class Event(Document):
     # TODO validate on this better
     data = DictField(required=True)
     time = FloatField(required=True)
-    meta = {'indexes': ['-descriptor', '-_id', '-uid'], 'db_alias': ALIAS}
+    meta = {'indexes': ['descriptor', '-time'], 'db_alias': ALIAS}
```

Comment:
Remember how ASCENDING vs DESCENDING made a big difference in terms of performance because of sorting the cursor? @tacaswell told us it was the bottleneck some time ago in event retrieval I think. 
From mongodb docs:
```
For compound indexes, sort order can matter in determining whether the index can support a sort operation.
```